src/Utilities/InputOutput.f90

```diff
@@ -1197,6 +1197,22 @@ subroutine get_ijk(nodenumber, nrow, ncol, nlay, irow, icol, ilay)
     !
     return
   end subroutine get_ijk
+  
+  !> @brief Function for string manipulation
+  !<
+  function padl(str, width) result(res)
```

Comment:
I think a more descriptive name should be used (maybe `str_pad_left`)

Reply:
@jdhughes-usgs, I'm all for descriptive names.  I installed the suggested name with bda8094